engines/sci/graphics/paint16.h

```diff
@@ -96,6 +98,16 @@ class GfxPaint16 {
 
 	// true means make EGA picture drawing visible
 	bool _EGAdrawingVisualize;
+
+	// The original KQ6WinCD interpreter saves the hires drawing information in a linked list. There are two use cases, one is redrawing the
+	// window background when recieving WM_PAINT messages (which is irrelevant for us, since that happens in the backend) and the other is
```

Comment:
```suggestion
	// window background when receiving WM_PAINT messages (which is irrelevant for us, since that happens in the backend) and the other is
```